tests/gatsby-starter-default/src/components/Collection.js

```diff
@@ -11,21 +11,15 @@ export const query = graphql`
     id
     title
     description
-    _components {
-      breadcrumbs {
-        items {
-          _id
-          _path
-          title
-        }
-      }
+    text {
+      react
     }
-    children {
-      ...Document
-      ...Event
-      ...File
-      ...NewsItem
-      ...SubFolder
+    items {
+      _id
+      _path
+      _type
+      description
+      title
```

Comment:
@ajayns This is the move from children to items look like; we no longer have deep GraphQL data, but only flat list of items.

Reply:
Pretty impressive, this reduces complexity from what I can see. And that's very convenient for developers!